metrics-exporter-prometheus/src/builder.rs

```diff
@@ -579,13 +579,16 @@ mod tests {
 
     use super::{Matcher, PrometheusBuilder};
 
+    static METADATA: metrics::Metadata =
+        metrics::Metadata::new(module_path!(), metrics::Level::INFO, Some(module_path!()));
```

Comment:
Eh, it's hard to see the value at this point, given that there's only three parameters needed. I guess, maybe, if we added something like `metrics::default_metadata!()`?

Avoiding needing to type out the `module_path!()` macro in the right place would be one thing, but I don't know if the complexity would be worth it to make it support a builder pattern (i.e. use the default level but set a custom target) over just having people call `Metadata::new` themselves... or providing builder pattern methods on `Metadata`.

Reply:
> providing builder pattern methods on Metadata

Regarding this, would you prefer it? I've been quite conservative with what we've added to `Metadata`. The `Metadata::new` obviously ties us to the current set of fields, if you anticipate perhaps adding something like [`Metadata::file`](https://docs.rs/tracing/latest/tracing/struct.Metadata.html#method.file) in the future then `Metadata::builder` is probably preferable?

Reply:
Yeah, let's just wait.

While the builder methods would make it easier to write a declarative macro (since we could just collect each build method fragment and smoosh them together at the end) there's still a little bit tied up in setting/specifying the defaults for the current field and it's not clear to me what the sharp edges, if any, will be with the current approach... so I think we probably just need to use it first.